docs/docs/guides/web3_config/index.md

```diff
@@ -198,4 +199,46 @@ console.log(web3.getContextObject().config)
   transactionPollingTimeout: 750000,
   ...
 */
-```
\ No newline at end of file
+```
+
+
+### defaultReturnFormat
```

Comment:
Think there should be a guide about defaultReturnFormat under upgrading from 1.x too because this PR is targetted towards users upgrading